### DISCUSSION TITLE
Turf Union Example Not Working For Me

### DISCUSSION THREAD

**Discussion by jamesrothwellhughes** [Turfjs/turf#2361] [General]
Hi, I am getting the following error when running the union example:

Cannot read properties of undefined (reading 'type')

Any suggestions (it's literally running the example code for the union function.

Regards,
James.

**Comment by rowanwins** [comment]
Hi @jamesrothwellhughes ,

You'll need to provide a snippet so we can see what you are doing - unfortunately there isn't enough detail in the question.

**Comment by Deveelop** [comment]
Hi @jamesrothwellhughes,
I seem not to understand your question also 
Provide a snippet please.
Thank you
